site/src/pages/docs/advanced-metrics.mdx

```diff
@@ -0,0 +1,73 @@
+# Collecting metrics
+
+Armeria has built-in support for collecting metrics both on the server and client side. This page describes how to enable this.
+
+The metric data is collected using the [Micrometer](https://micrometer.io/) library. Please consult its documentation for information on how to expose the collected metrics to various monitoring systems (JMX, Prometheus, etc)
```

Comment:
Global comment: Please break the line if the line length exceeds 112.

Reply:
How about:

    ... systems, such as [Prometheus](...), [Dropwizard Metrics](...) and [Datadog](...).